demo/main.js

```diff
@@ -39,19 +42,24 @@ class RootView extends React.Component {
         );
     }
 
-    getEditor({field, operator, value, onChange}) {
-        if (field !== 'isDev' || operator !== '=') {
-            return null;
-        }
+    customValueEditor() {
+        class MyCheckbox extends React.Component {
+            render() {
+                const hasValue = !!this.props.value;
+                return (
+                    <span>
+                        <input type="checkbox"
+                               value={hasValue}
+                               onChange={event=>this.props.handleOnChange(event.target.checked)}/>
+                    </span>
+                );
+            }
 
-        const hasValue = !!value;
-        return (
-            <span>
-            <input type="checkbox"
-                   value={hasValue}
-                   onChange={event=>onChange(event.target.checked)}/>
-        </span>
-        );
+            static shouldRender(props) {
+                return props.field === 'isDev' && props.operator === '=';
```

Comment:
Don't think we should have this. How about using the `render` itself to decide whether to render or not? Simplifies the contract of a custom control